### PR TITLE
ENH: Series.append now has ignore_index kw

### PR DESCRIPTION
**Pull request by sinhrks**
- [x] tests added / passed
- [x] passes `git diff upstream/master | flake8 --diff`
- [x] whatsnew entry

not sure if there is any reason we haven't supported...

### CODE REVIEW

File: pandas/tests/series/test_combine_concat.py [hunk 1]
```diff
@@ -39,6 +39,21 @@ def test_append_many(self):
         result = pieces[0].append(pieces[1:])
         assert_series_equal(result, self.ts)
 
+    def test_append_duplicates(self):
+        s1 = pd.Series([1, 2, 3])
+        s2 = pd.Series([4, 5, 6])
```
Comment (jreback) [inline pandas/tests/series/test_combine_concat.py]: can you add a simultaneous example with ``pd.concat`` (same results)

**Comment by jreback** [comment]
lgtm. just minor test comment.

**Comment by sinhrks** [comment]
updated test, and now green.

**Comment by jreback** [comment]
thanks!

**Pull request closed without merging**